AAI/AAIConnectProfile.md

```diff
@@ -669,18 +710,26 @@ Payload:
 
 -   `addtional claims`: OPTIONAL. Any other additional non-GA4GH claims are allowed. This specification does not dictate the format of other claims.
 
+<hr style="width: 10em; margin: 2em auto;"/>    
+
 #### Claims sent to Data Holder by a Broker via Token or UserInfo Endpoint
 
+##### Claims via UserInfo Endpoint
+
+The endpoint MAY use `application/json` or `application/jwt`. It is RECOMMENDED that if desiring to return a JWT, a
```

Comment:
This section most repeats what is already in [OIDC Core section 5.3.2 "Successful UserInfo Response"](https://openid.net/specs/openid-connect-core-1_0.html#UserInfoResponse) so I think just the first paragraph is actually needed.

Reply:
@martin-kuba I removed most of this section, but I kept the bullet point on `<ga4gh-spec-claims>` and made it more specific (so it refers to `ga4gh_passport_v1`). WDYT?